.github/workflows/sycl-windows-build.yml

```diff
@@ -78,7 +81,7 @@ permissions: read-all
 jobs:
   build:
     name: Build + LIT
-    runs-on: [Windows, build]
+    runs-on: [Windows, build-e2e]
```

Comment:
@uditagarwal97 @aelovikov-intel 
Just FYI about this part, it was my suggestion that we try to run all Windows build jobs, not just e2e test building, on the `build-e2e` label machines because they're much faster.

Reply:
I'll do that now.